static/js/components/ViewProfileDisplay.js

```diff
@@ -0,0 +1,210 @@
+                    </div>
+                  </div>
+                ) : null}
+                {countries && currentUser.legal_address.state_or_territory ? (
+                  <div className="row profile-row">
+                    <div className="col profile-label">
+                      State/Province/Region
+                    </div>
+                    <div className="col">
+                      {
+                        find(
+                          propEq(
+                            "code",
+                            currentUser.legal_address.state_or_territory
+                          ),
+                          find(
+                            propEq("code", currentUser.legal_address.country),
+                            countries
+                          ).states
+                        ).name
+                      }
+                    </div>
+                  </div>
+                ) : null}
+                <div className="divider" />
+                <div className="row profile-row">
+                  <div className="col profile-label">Gender</div>
+                  <div className="col">
+                    {fromPairs(GENDER_CHOICES)[currentUser.profile.gender]}
+                  </div>
+                </div>
+                <div className="row profile-row">
+                  <div className="col profile-label">Year of Birth</div>
+                  <div className="col">{currentUser.profile.birth_year}</div>
+                </div>
+                <div className="row profile-row">
+                  <div className="col profile-label">Company</div>
+                  <div className="col">{currentUser.profile.company}</div>
+                </div>
+                <div className="row profile-row">
+                  <div className="col profile-label">Job Title</div>
+                  <div className="col">{currentUser.profile.job_title}</div>
+                </div>
+                <div className="form-group dotted" />
+                <div className="row profile-row">
+                  <div className="col profile-label">Industry</div>
+                  <div className="col">{currentUser.profile.industry}</div>
+                </div>
+                <div className="row profile-row">
+                  <div className="col profile-label">Job Function</div>
+                  <div className="col">{currentUser.profile.job_function}</div>
+                </div>
+                <div className="row profile-row">
+                  <div className="col profile-label">Company Size</div>
+                  <div className="col">
+                    {
+                      fromPairs(EMPLOYMENT_SIZE)[
+                        currentUser.profile.company_size
+                      ]
+                    }
+                  </div>
+                </div>
+                <div className="row profile-row">
+                  <div className="col profile-label">
+                    Years of Work Experience
```

Comment:
Not sure if this matters but in the design it's just `Years of Experience`

Reply:
We have been going back and forth on this. The current thinking is "Years of Work Experience" (from comment #4 on https://projects.invisionapp.com/d/main#/console/19928472/417192799/comments/132002938)

Reply:
leaving as is for now